### DISCUSSION TITLE
Why I can't see my recent Articles ?

### DISCUSSION THREAD

**Discussion by wilonweb** [nunocoracao/blowfish#1436] [Q&A] [answered]
Hello,

I am trying to display a list of articles on my homepage as well as on one of my taxonomy pages:
- mydomain.com/blog
- mydomain.com/tuto

I placed a `'list'` shortcode in my `content\index.md`. I see the default title 'Recents', but the list of my other articles does not appear.
```
---
title: "Welcome to Blowfish! :tada:"
description: "This page was built using the Blowfish theme for Hugo."
---

Hello World

<h3>bonjour</h3>

{{< list limit=2 >}}

```

This is the structure of my content folder:
```
my-awesome-article
    featured.svg
    index.md
my-article-2
    index.md
index.md
```

And in `config/_default/params.toml`:
```
[homepage]
  layout = "background" 
  homepageImage = "blowfish_banner.png" # used in: hero, and card
  showRecent = true
  showRecentItems = 6
  showMoreLink = true
  showMoreLinkDest = "blog"
  cardView = true
  cardViewScreenWidth = false
  layoutBackgroundBlur = true # only used when layout equals background
```

For this, I created `config/_default/taxonomies.toml`:
```
tuto = "tuto"
blog = "blog"
```

And this is `content/index.md`:
```
---
title: "Welcome to Blowfish! 🎉"
description: "This page was built using the Blowfish theme for Hugo."
---

Hello World

<h3>bonjour</h3>
{{< article link="/my-awesome-article" >}}
```

I also created `config/_default/taxonomies.toml`:
```
tuto = "tuto"
blog = "blog"
```

And I indeed have pages at the URLs
- mydomain.com/tuto
- mydomain.com/blog

However, I do not see any articles. Here is the front matter of one of my articles:
```
---
title: "Mon Super Article Wilou"
description: "This is a demo of adding content to the homepage."
date: 2021-03-17
categories: ["blog", "tuto"]
showHero: true
draft: false
---
```
My question is:
How can I display a list of recent articles on my homepage?

And how can I display articles that belong to the 'blog' taxonomy on my blog page, and similarly for those belonging to the 'tuto' taxonomy?

You can check my code here : 
https://github.com/wilonweb/blowfish.git

**Comment by nunocoracao** [comment, marked as answer]
Hey, here are some changes that will probably solve it
- change the folder structure
- change the main index to _index.md otherwise the content folder is identified as an article not as a list of articles
- change the menu configs... 

Probably only need the index change but these are the changes I made to get it to work

![Screenshot 2024-04-28 at 22 54 44](https://github.com/nunocoracao/blowfish/assets/3803196/22b9a248-846d-49b9-81b1-72b809e86dee)
![Screenshot 2024-04-28 at 22 54 56](https://github.com/nunocoracao/blowfish/assets/3803196/0eb29a5f-08f8-4269-ac9c-87ba4a5ed28c)
![Screenshot 2024-04-28 at 22 55 06](https://github.com/nunocoracao/blowfish/assets/3803196/d15428a6-51dc-4644-9e2a-01ca00424417)
